.github/workflows/publish-node.yml

```diff
@@ -228,7 +228,6 @@ jobs:
           image: ${{ matrix.settings.docker }}
           options: -v ${{ env.HOME }}/.cargo/git:/root/.cargo/git -v ${{ env.HOME }}/.cargo/registry:/root/.cargo/registry -v ${{ github.workspace }}:/build -w /build
           run: >-
-            yarn global add wasm-pack &&
```

Comment:
https://github.com/swc-project/swc/blob/c746611f262d8b3fbac1c29c23f6c2a4c5f3ce2b/.github/workflows/publish-node.yml#L219-L223

We have installed wasm-pack above these lines.
Is it necessary to install it again in docker step?
@kdy1

Reply:
I don't think so, seems like a mistake